client/src/services/preview-helper.service.ts

```diff
@@ -33,4 +44,37 @@ export default class PreviewHelperService {
             relatedPatents: citedPatents,
         };
     }
+
+    /**
+     * Helper to get the author name and the invented patents for node preview
+     * @param node
+     * @param patents
+     */
+
+    static getAuthorPreview(node: NodeInfo, patents: Patent[]): NodePreview {
+        const authorPatents = patents.filter((patent) => patent.inventors?.includes(node.id));
+        return {
+            id: node.id,
+            title: `${node.id}`,
+            type: node.type,
+            subTitle: 'Inventor of',
```

Comment:
It's possible we should standardize the app to always use either 'Author' or 'Inventor'.

Reply:
I also was confused at this point. i checked our product and in the detailed patent preview is also written, inventors. That is why I thought to add. But maybe inventors for patents is a more appropriate English word than the author. What do you think @micahh2?
![image](https://user-images.githubusercontent.com/34209362/141116070-3a8e0198-34b2-42cc-bc2e-2dc65d783893.png)

Reply:
I think (personally) that Author sounds more general and is more likely to be correct. However, I don't really know what the correct nomenclature is for the world of patents. Either is probably fine for our purposes.
The conflicting UI I was thinking of is here:
![image](https://user-images.githubusercontent.com/646155/141118552-27583333-9fba-45fa-bbd6-f26cdee506fa.png)

Reply:
I agree. When working on retrieving that information I saw that OPS is calling them Inventors and Applicants. Applicanty are mostly companies, but it can be that a single person is also the applicant and the inventor. So I wasn't really sure which naming to pick.

In the end we either can decide to stick with Authors & Companies and rename those references in the rest of the app or we can rename `authors` to `inventors` (which would be more accurate) and `companies` to `applicants`

Reply:
I think it makes a lot of sense to mirror what OPS is using. Maybe we make that another card.

Reply:
> I agree. When working on retrieving that information I saw that OPS is calling them Inventors and Applicants. Applicanty are mostly companies, but it can be that a single person is also the applicant and the inventor. So I wasn't really sure which naming to pick.
> 
> In the end we either can decide to stick with Authors & Companies and rename those references in the rest of the app or we can rename `authors` to `inventors` (which would be more accurate) and `companies` to `applicants`

I have checked the google patent and they also use inventors instead of authors. I think this is the correct name.
![image](https://user-images.githubusercontent.com/34209362/141120038-6f7dac54-5bc1-47be-be6a-4f8f7c470e61.png)
Source: https://patents.google.com/patent/US4991350?oq=cat

Reply:
I have updated the inventor to Author to be like in the option many(toggle author).